README.md

```diff
@@ -42,6 +42,15 @@ packages.
 
 See more example code at https://github.com/insomniacslk/exdhcp
 
+# Fuzzing 
```

Comment:
Do we really need this? This is not different to what the `go-fuzz` docs mention.

Reply:
ReMoved